src/main/scala/cromwell/engine/backend/local/LocalBackend.scala

```diff
@@ -143,6 +144,25 @@ case class LocalBackend(actorSystem: ActorSystem) extends Backend with SharedFil
     s"docker run --rm -v ${backendCall.callRootPath.toAbsolutePath}:$callPath -i $image"
   }
 
+  /**
+    * If using Mac OS X with Docker Machine, by default Cromwell can only use the -v flag to 'docker run'
+    * if running from somewhere within the user's home directory.  If not, then -v will mount to the
+    * virtual machine that Docker Machine creates.  This ends up bubbling up to Cromwell when it can't find
+    * an RC file in the call's output directory (because it's actually on the virtual machine).
+    *
+    * This function will at least detect this case and log some WARN messages.
+    */
+  private def detectDockerMachinePossibleMisusage(backendCall: BackendCall, logger: WorkflowLogger) = {
```

Comment:
Though if we're okay with the possibility of a false positive, or if we can figure out a way to query virtualbox for volumes that are mounted to Mac OS, this error could be reported sooner